src/GenericForm.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import './App.css';
+import Input from './Input.js';
+import Textarea from './Textarea.js';
+import Title from './Title.js';
+import Radio from './Radio.js';
+import Select from './Select.js';
+import Password from './Password.js';
+
+class GenericForm extends Component {
+
+    render() {
+
```

Comment:
Please remove useless empty line rows and update indentation with 2 spaces in all files.